dbms/src/Core/iostream_debug_helpers.cpp

```diff
@@ -95,12 +95,14 @@ std::ostream & operator<<(std::ostream & stream, const Connection::Packet & what
     return stream;
 }
 
+#if 0
```

Comment:
You should provide a comment.
I don't understand, what does it means, what is the motivation...